packages/react-router-dom/index.tsx

```diff
@@ -1442,9 +1443,10 @@ function useSubmitFetcher(
         body,
         formMethod: options.method || (method as HTMLFormMethod),
         formEncType: options.encType || (encType as FormEncType),
+        persist,
```

Comment:
The `useFetcher({ persist })` value is proxied through to the `router` at `router.fetch` time, not at `useFetcher`-creation time.  This makes it easier to target submitting fetchers only